definitions/firefox_desktop.toml

```diff
@@ -1073,26 +1073,178 @@ description = """
 Count of impressions of Sponsored Pocket content.
 """
 
+[metrics.all_tile_impressions_pos1_2]
+data_source = "newtab_visits_topsite_tile_interactions"
+select_expression = """COALESCE(
+        SUM(CASE
+          WHEN (
+            topsite_tile_interactions.sponsored_topsite_tile_impressions > 0 AND topsite_tile_interactions.topsite_tile_position < 2
```

Comment:
We can make these more concise:
`CASE WHEN
  topsite_tile_interactions.topsite_tile_position < 2
    THEN COALESCE(topsite_tile_interactions.sponsored_topsite_tile_impressions, 0) +  COALESCE(topsite_tile_interactions.organic_topsite_tile_impressions, 0)
  ELSE 0
  END`